cmd/pod-scaler/admission.go

```diff
@@ -243,16 +243,23 @@ func preventUnschedulable(resources *corev1.ResourceRequirements) {
 	if resources.Requests == nil {
 		return
 	}
-	if _, ok := resources.Requests[corev1.ResourceCPU]; !ok {
-		return
+
+	if _, ok := resources.Requests[corev1.ResourceCPU]; ok {
+		// 10 CPU is a ballpark number. Current build farm nodes have 16 CPU so pods get unschedulable
+		// around 14 CPU
+		// TODO(DPTP-2525): Make configurable and perhaps cluster-specific?
```

Comment:
IMHO this could be done in this PR. At least make it configurable and as a follow-up, we can determine the desired limit per cluster.

Reply:
yeah, I suppose it would be nice if both the CPU and memory caps were configurable